cli/cmd/initnodes.go

```diff
@@ -416,9 +411,14 @@ func gethInit(ctx context.Context, cfg InitConfig, dir string) error {
 	{
 		image := "ethereum/client-go:" + geth.Version
 		stateScheme := "path"
+		if cfg.NodeSnapshot {
```

Comment:
when restoring node snapshots, we don't need to do geth init, so you shouldn't need to do any of this

Reply:
There is some inconsistency with staging that I am currently investigating, so that is why I put it here, but will be removed.